tests/integration/test.local_docs.js

```diff
@@ -86,17 +86,14 @@ adapters.forEach(function (adapter) {
       var db = new PouchDB(dbs.name);
       var doc = {_id: '_local/quux'};
       return db.put(doc).then(function (res) {
-        res.rev.should.equal('0-1');
```

Comment:
This is the only test I am worried about, I dont like depending on `NEXT` etc too much, and right now the `indexeddb` adapter doesnt copy the revision number generation that current pouch / couch does although it behaves the same way otherwise